### DISCUSSION TITLE
Submit Bromite to official F-Droid store

### DISCUSSION THREAD

**Discussion by mat-l** [bromite/bromite#1028] [General]
Since some trouble with Google's Android WebViewer some days ago I was searching a while for an open alternative. I just came across Bromite and Bromite SystemWebView quite accidently due to a hint.
Submitting your great apps to the official F-Droid Store would increase visibility to the user who want to take a step back from Google.

Are there any plans for this topic?
Best regrads, keep this project up and stay healthy!

**Comment by bat999** [comment]
Hi
You need to add the Bromite repo to the F-Droid app.
Look here ---> https://www.bromite.org/fdroid

**Reply by TheAkashicTraveller** [reply]
woosh

**Reply by mat-l** [reply]
@bat999 I know, I downloaded it from the external repo.

The question is, why not publish it directly in the original F-Droid Repo, which is easier accessible for other people and the app has better visibility.

**Reply by Canvas02** [reply]
I'm pretty sure that when you want your app on the official f-droid repo they are the ones who build the apk not you

So you can't just upload an apk, no you need to tell the f-droid build bot that you want an apk 

> I don't have any experience in putting apps on the official f-droid repo so I may be wrong 

> All the info I know is from [Iceraven's attempt](https://github.com/fork-maintainers/iceraven-browser/issues/26) on putting there app on the official f-droid repo

**Reply by csagan5** [reply]
See https://github.com/bromite/bromite/blob/master/FAQ.md#is-bromite-on-f-droid

You should search the issue tracker for more information.

**Reply by Canvas02** [reply]
I know that bromite is already has an f-droid repo and has no plans to be added on the official one, this guy asked about why not
